package.json

```diff
@@ -10,6 +10,9 @@
   "license": "MIT",
   "dependencies": {},
   "devDependencies": {
-    "danger": "^10.4.0"
+    "danger": "^10.4.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "webpack": "^4.44.1"
```

Comment:
Added these dependencies to force the package-lock.json to have lots of changes, they are not used by the danger recipe.